integrationExamples/gpt/gdpr_hello_world.html

```diff
@@ -82,37 +82,37 @@
       <script>
         var PREBID_TIMEOUT = 700;
         var FAILSAFE_TIMEOUT = 2500;
-  
```

Comment:
gdpr_hello_world.html shouldn't be included in this commit.  Please revert the changes to it

Reply:
i revert changes, but git show that i remove line and than end line again. it's a problem ?